react/package.json

```diff
@@ -35,11 +35,18 @@
     "start": "npm run build && craco start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test:coverage": "cross-env CI=true yarn test --env=jsdom --coverage",
```

Comment:
cross-env is an outdated package which you can and probably should drop

Reply:
The medium article you linked also uses cross-env. Do you know about any other packages I can use to account for different environments? `CI=true yarn test --env=jsdom --coverage` works for a Mac but not a PC.